MergerService/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace MergerService.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
+    {
+        private static readonly string[] Summaries = new[]
```

Comment:
meaningful name

Reply:
that's the basic service template.
I kept it as its good example of service structure and should be removed when the service is added.